app/jenkins-for-jira-ui/src/components/ConnectionPanel/NotConnectedState.tsx

```diff
@@ -96,6 +102,29 @@ const NotConnectedState = ({
 							/>
 						</>
 					)}
+
+			{
+				showRetryServerDelete &&
+				(isDeletingServer && jenkinsServer.uuid === serverToDeleteUuid) &&
+					<JenkinsModal
+						dataTestId={DELETE_MODAL_TEST_ID}
+						server={jenkinsServer}
+						modalAppearance='danger'
+						title={`An error occurred while deleting your connection to ${jenkinsServer?.name}`}
+						body={[
```

Comment:
Could you apply the logic in the above comment to the rest of these `body` props?